pkgs/top-level/all-packages.nix

```
@@ -8974,6 +8974,8 @@ with pkgs;
 
   gtdialog = callPackage ../development/libraries/gtdialog { };
 
+  gtkclipblock = callPackage ../tools/misc/gtkclipblock { stdenv = gcc13Stdenv; };
```

I suggest to use the new by-name hierarchy.
```suggestion
  gtkclipblock = callPackage ../by-name/gt/gtkclipblock/package.nix { 
   stdenv = gcc13Stdenv; 
  };
```

---

Done.